code/modules/mob/living/carbon/human/species/station/station.dm

```diff
@@ -308,6 +308,111 @@
 /datum/species/skrell/can_breathe_water()
 	return TRUE
 
+/datum/species/zaddat
+	name = SPECIES_ZADDAT
+	name_plural = "Zaddat"
+	icobase = 'icons/mob/human_races/r_zaddat.dmi'
+	deform = 'icons/mob/human_races/r_zaddat.dmi'
+	unarmed_types = list(/datum/unarmed_attack/stomp, /datum/unarmed_attack/kick, /datum/unarmed_attack/punch)
+	brute_mod = 1.15
+	burn_mod =  1.15
+	toxins_mod = 1.5
+	flash_mod = 2
+	flash_burn = 15 //flashing a zaddat probably counts as police brutality
+	metabolic_rate = 0.7 //did u know if your ancestors starved ur body will actually start in starvation mode?
+	gluttonous = 1
+	taste_sensitivity = TASTE_SENSITIVE
+	num_alternate_languages = 3
+	secondary_langs = list(LANGUAGE_ZADDAT, LANGUAGE_UNATHI)
+	assisted_langs = list(LANGUAGE_EAL, LANGUAGE_TERMINUS, LANGUAGE_SKRELLIANFAR, LANGUAGE_ROOTLOCAL, LANGUAGE_ROOTGLOBAL, LANGUAGE_VOX, LANGUAGE_SOL_COMMON, LANGUAGE_AKHANI, LANGUAGE_SIIK, LANGUAGE_GUTTER) //has trouble talking low
+	name_language = LANGUAGE_ZADDAT
+	species_language = LANGUAGE_ZADDAT
+	health_hud_intensity = 2.5
+
+	minimum_breath_pressure = 20 //have fun with underpressures. any higher than this and they'll be even less suitible for life on the station
+
+	economic_modifier = 3
+
+	min_age = 16
+	max_age = 90
+
+	blurb = "The Zaddat are an Unathi client race only recently introduced to SolGov space. Having evolved on the high-pressure and post-apocalyptic world of Xohok, Zaddat require an environmental suit called a Shroud to survive in usual planetary and station atmospheres. Despite these restrictions, worsening conditions on Xohok and the blessing of the Moghes Hegemony have lead the Zaddat to enter human space in search of work and living space."
+
+	hazard_high_pressure = HAZARD_HIGH_PRESSURE + 500  // Dangerously high pressure.
+	warning_high_pressure = WARNING_HIGH_PRESSURE + 500 // High pressure warning.
+	warning_low_pressure = 300   // Low pressure warning.
+	hazard_low_pressure = 220     // Dangerously low pressure.
+	safe_pressure = 400
+	poison_type = "nitrogen"      // technically it's a partial pressure thing but IDK if we can emulate that
+
+	genders = list(FEMALE, NEUTER)
```

Comment:
Why is it these two, again? Also, if you don't want other creatures to be able to tell what they are, there's a var for that.

Reply:
their gender-shaped clades are "polyp-producing female" and "other" for all practical purposes. They're not ambigious_gender because females have obviously visible vestigial arms on their torsos.

Reply:
...should actually be plural, though